program-runtime/src/sysvar_cache.rs

```diff
@@ -56,6 +57,16 @@ impl SysvarCache {
         self.epoch_schedule = Some(Arc::new(epoch_schedule));
     }
 
+    pub fn get_epoch_rewards(&self) -> Result<Arc<EpochRewards>, InstructionError> {
+        self.epoch_rewards
+            .clone()
+            .ok_or(InstructionError::UnsupportedSysvar)
```

Comment:
nit: `UnsupportedSysvar` may not be the best error, since it *is* supported, just not at the time of calling. Maybe something like `UnavailableSysvar` would be clearer. note that I don't feel too strongly about this

Reply:
Are you suggesting to add a new variant of UnavailableSysvar to InstructionError Enum? 

I see the following comments in the InstructionError Enum.
```
/// Reasons the runtime might have rejected an instruction.
///
/// Members of this enum must not be removed, but new ones can be added.
/// Also, it is crucial that meta-information if any that comes along with
/// an error be consistent across software versions.  For example, it is
/// dangerous to include error strings from 3rd party crates because they could
/// change at any time and changes to them are difficult to detect.
```
Adding new "UnavailableSysvar` will definitely break ABI compatibility. What other impact will this change have?